apps/token-manager/app/src/script.js

```diff
@@ -1,6 +1,6 @@
 import Aragon, { events } from '@aragon/api'
+import { addressesEqual }  from '@aragon/ui'
```

Comment:
Ah... this may be a problem @bpierre.

This is still run in the context of a browser, but I think we should aim for this script to be runnable in a node / etc. environment. AFAIK nothing else used by the script should cause an issue.

From what I remember, importing `@aragon/ui` may force a browser context?

Reply:
It should be fine, aragonUI is now compatible with non-DOM environments: https://github.com/aragon/aragon-ui/pull/699

But another thing to take care of would be to ensure that the entire library doesn’t get included in script.js. That would require using [`--experimental-scope-hoisting`](https://en.parceljs.org/cli.html#enable-experimental-scope-hoisting/tree-shaking-support), but we were having issues with it IIRC.

What would you guys think of inlining the function in every script.js using it, until we move to Parcel 2 or another bundler with a good support for tree shaking? https://github.com/aragon/aragon-ui/blob/master/src/utils/web3.js#L82-L92